Pre_processing.py

```
@@ -0,0 +1,74 @@
+from pyspark.ml.feature import Tokenizer
+from pyspark.ml.feature import StopWordsRemover
+from pyspark.ml.feature import HashingTF, IDF
+
+
+# COMMAND ----------
+
+class PreProcess:
+    #Contains various functions to prepare the data for modelling
+    def load_data(self,filepath):
+         #Low the data from the filepath
+        df_out = spark.read.csv(filepath,header = False)
```

Ruff

Undefined name `spark`

(F821)